cami.c

```diff
@@ -511,15 +574,19 @@ void ami_set_debug(struct ami_session *ami, int fd)
 
 int ami_set_debug_level(struct ami_session *ami, int level)
 {
-	int old_level = ami->debug_level;
+	int old_level = ami ? ami->debug_level : ami_debug_level;
```

Comment:
Please update the documentation in cami.h for the public functions that have been changed.

e.g. `\param ami The AMI session. If NULL, this will set the initial debug level for sessions, used only during session creation (inside ami_connect)` or something like that. Also add they may need to call the function again for the individual session to actually set the debug level at that point. Same thing for `ami_set_debug`